processing/src/main/java/io/druid/query/aggregation/MetricManipulatorFns.java

```diff
@@ -21,39 +21,45 @@
  */
 public class MetricManipulatorFns
 {
-  public static MetricManipulationFn identity()
+  private static final MetricManipulationFn IDENTITY_INSTANCE = new MetricManipulationFn()
   {
-    return new MetricManipulationFn()
+    @Override
+    public final Object manipulate(final AggregatorFactory factory, final Object object)
```

Comment:
It doesn't, I probably meant to put the final on the object instead of the overridden method and just didn't realize the mixup.